package.json

```diff
@@ -39,6 +40,7 @@
     "portfinder": "~1.0.10",
     "request": "~2.87.0",
     "requestretry": "~2.0.2",
+    "sauce-connect-launcher": "^1.3.1",
```

Comment:
Let's use `~` or pin this version to be safe

Reply:
Just clarify the difference of `~` and `^` here to make sure this is what we really want to do:
`~`: we limit the change only to `1.3.x`
`^`: we limit the change to `1.x`
Have updated.